terraform/provisioning/ubuntu.bash

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly root=$(dirname "${BASH_SOURCE[0]}")
+
+# shellcheck source=/dev/null
+source "${root}/common.bash"
+
+install_docker() {
+  # https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+  # Try to avoid 'Splitting up ...InRelease into data and signature failed'
+  # errors by trying again and again :( (Usually works on second attempt)
```

Comment:
what is this error? is there an URL about it?

Reply:
I don't really know and haven't found an answer as to why. I'm guessing it's some caching problem with particular (AWS?) images/mirrors?

Reply:
I guess it's ok for now and we can revisit later.